{
  "Type": "System.OperationCanceledException",
  "Message": "The operation was canceled.",
  "Source": "System.Private.CoreLib",
  "StackTrace": "   at System.Threading.CancellationToken.ThrowOperationCanceledException()\r\n   at System.Threading.CancellationToken.ThrowIfCancellationRequested()\r\n   at Speckle.Sdk.Serialisation.V2.Receive.ObjectLoader.GetAndCache(String rootId, DeserializeProcessOptions options) in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\src\\Speckle.Sdk\\Serialisation\\V2\\Receive\\ObjectLoader.cs:line 66\r\n   at Speckle.Sdk.Serialisation.V2.Receive.DeserializeProcess.Deserialize(String rootId) in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\src\\Speckle.Sdk\\Serialisation\\V2\\Receive\\DeserializeProcess.cs:line 85\r\n   at Speckle.Sdk.Serialization.Tests.CancellationTests.<>c__DisplayClass4_0.<<Cancellation_Receive_Cache>b__0>d.MoveNext() in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\tests\\Speckle.Sdk.Serialization.Tests\\CancellationTests.cs:line 113\r\n--- End of stack trace from previous location ---\r\n   at Xunit.Assert.RecordExceptionAsync(Func`1 testCode) in /_/src/xunit.assert/Asserts/Record.cs:line 114"
}
